jscomp/ml/ctype.ml

```diff
@@ -3904,6 +3904,11 @@ let subtypes = TypePairs.create 17
 let subtype_error env trace =
   raise (Subtype (expand_trace env (List.rev trace), []))
 
+let extract_concrete_typedecl_opt env t = 
+  match extract_concrete_typedecl env t with 
+  | v -> Some v 
+  | exception _ -> None
```

Comment:
Do you know what exception is raised here?
If you do, it seems safer to catch the expected one.

Reply:
Yes I do. Will update.